src/BatteryStateOfChargeService.h

```diff
@@ -1,6 +1,14 @@
 #pragma once
 
 #include "I_BatteryStateOfChargeService.h"
+#include "BatteryStateOfChargeService.h"
+#include <QDebug>
+#include <QFile>
+#include <QString>
+#include <QTextStream>
```

Comment:
The `#include`s on lines 4-8 are not necessary since you've already included them in the `.cpp` and you are not using them anywhere here.